tests/templates/pvc-pvdir.yaml

```diff
@@ -27,6 +27,15 @@ apiVersion: v1
 metadata:
   name: hostpath-test-pod-pvdir
 spec:
+  affinity:
```

Comment:
Why is this affinity rule needed? Are we saying that in a multinode cluster hostpath-provider will not work?

Reply:
This is for the same reason as above, to ensure that that this pod is on the same node as that of hostpath pod.